### PR TITLE
get-covid-stats JavaScript version

### PR DESCRIPTION
**Pull request by mendoza**
I implemented the get-covid-stats function for NodeJS, with README and package.json
mentioned on issue appwrite/appwrite#1923

# Executed function
image
# Execution log
image

### CODE REVIEW

File: nodejs/get-covid-stats/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1430 @@
+{
+  "name": "cloud-functions-demo-nodejs",
```
Comment (christyjacob4) [inline nodejs/get-covid-stats/package-lock.json:2]: You need to remove this file from the PR

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Also please share screenshots of a successful execution in the Appwrite Dashboard with the log message :+1:

File: nodejs/get-covid-stats/package-lock.json (hunk 1 above)
Comment (mendoza) [reply nodejs/get-covid-stats/package-lock.json:2]: i removed the file i'm about to commit

File: nodejs/get-covid-stats/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+$ tar -zcvf code.tar.gz get-covid-stats
+```
+
+* Upload the tarfile to your Appwrite Console and use the following entrypoint command
+
+```bash
+node index.js
+```
+## 🎯 Trigger
+Can be triggered from manually from the Appwrite Console.
```
Comment (christyjacob4) [inline nodejs/get-covid-stats/README.md:40]:
```suggestion
Can be triggered manually from the Appwrite Console.
```

File: nodejs/get-covid-stats/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# 🦠 Function to fetch COVID-19 statistics
+Javascript function that prints today's COVID statistics for a country or for the whole world, with a helper function for the format.
+
+## 📝 Environment Variables
+When running function, send the country code as function data.
+
+_The country must be in ISO2 format. For example `US`, `HN` etc. If it's not sent, global stats will be returned._
+
```
Comment (christyjacob4) [inline nodejs/get-covid-stats/README.md:8]: Let's use the same format for specifying the environment variables as the other examples. 
https://github.com/appwrite/demos-for-functions/tree/master/nodejs/welcome-email#-environment-variables

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: nodejs/get-covid-stats/README.md (hunk 3 above)
Comment (mendoza) [reply nodejs/get-covid-stats/README.md:8]: will do

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: nodejs/get-covid-stats/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# 🦠 Function to fetch COVID-19 statistics
+Javascript function that prints today's COVID statistics for a country or for the whole world, with a helper function for the format.
+
+## 📝 Environment Variables
+Go to Settings tab of your Cloud Function. Add the following environment variables.
+- APPWRITE_FUNCTION_DATA - Country code in ISO2 format, For example `US`, `HN` etc.
```
[the remark this answers is not in the document]
Comment (mendoza) [reply nodejs/get-covid-stats/README.md:7]: I made a longer but better paragraph on the environment variable section, to explain where we can set it too.

**Review by Meldiron — APPROVED** [review]
Looks good to me

**Comment by Meldiron** [comment]
THANK YOU! All changes merged 🥳

Please reach out to `eldadfux` on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**